src/Sema.zig

```diff
@@ -25333,6 +25333,7 @@ fn coerceExtra(
                     } };
                     break :pointer;
                 }
+                if (dest_ty.isSlice()) break :to_anyopaque;
```

Comment:
I actually did have it there at first, but moved it so that the user could get the additional note about the double pointer:
`main2.zig:4:9: note: cannot implicitly cast double pointer '[]*u32' to anyopaque pointer '[]anyopaque'` 

My logic is this lets them know about at least something more about why they can't do this.